src/Client.php

```diff
@@ -25,295 +26,233 @@ class Client
      * Creates a new client instance with an API Key and API Secret
      * You can obtain these from the Communicacie, who will register your app in Lassie
      *
-     * @param string $key
-     * @param string $secret
+     * @param string $token static access token for Directus
      */
-    public function __construct(string $key, string $secret)
+    public function __construct(string $token, string $base_url = "https://admin.eshdavinci.nl")
     {
         $stack = new HandlerStack();
         $stack->setHandler(new CurlHandler());
-        $stack->push(KeyMiddleware::addData($key, $secret));
         $this->guzzleClient = new GuzzleClient([
-        'base_uri' => 'https://davinci.lassie.cloud/',
-        'timeout'  => 30.0,
-        'handler' => $stack
-    ]);
+            "base_uri" => $base_url,
+            "timeout" => 4.0,
+            "headers" => [
+                "Authorization" => "Bearer $token",
+                "Content-Type" => "application/json",
+            ]
+        ]);
     }
 
-    /**
-     * Checks the returned body for an error code
-     */
-    private function checkForError($body)
+    /** Checks the returned body for an error code */
+    private function checkForError($response): void
     {
-        if (isset($body['status_code'])) {
-            // An error is detected
-            $status = $body['status_code'];
-            if ($status === 403) {
-                throw new PermissionDeniedException($body['error']);
-            } elseif ($status === 404) {
-                throw new NotFoundException($body['error']);
-            }
-        }
+        $status_code = $response->getStatusCode();
+        if ($status_code % 200 < 100)
+            return;
+        $data = json_decode($response->getBody(), true);
+        var_dump($data);
+        if($status_code === 404)
```

Comment:
You should adopt some sort of coding style here, can you use `php-cs-fixer`? Then it's automatically correct.